src/main/java/TaskManager.java

```diff
@@ -0,0 +1,89 @@
+public class TaskManager {
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+
+    public void greet() {
+        System.out.println("  ____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("  ____________________________________________________________");
+    }
+
+    public void bye() {
+        System.out.println("  ____________________________________________________________");
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("  ____________________________________________________________");
+    }
+
+    public void addTask(String line, String commandWord) {
+        separatingCommands(line, commandWord);
+        System.out.println("  ____________________________________________________________");
+        System.out.println("\tAdded:");
+        System.out.println("\t  " + tasks[taskCount].getType() + tasks[taskCount].getStatus()
+                + tasks[taskCount].getDescription() + tasks[taskCount].getAddedInfo());
+        taskCount++;
+        System.out.println("\tNow you have " + taskCount + " tasks in the list");
+        System.out.println("  ____________________________________________________________");
+    }
+
+    private void separatingCommands(String line, String commandWord) {
+        if (commandWord.equals("todo")) {
+            createTodo(line);
+        } else if (commandWord.equals("deadline")) {
+            createDeadline(line);
+        } else if (commandWord.equals("event")) {
+            createEvent(line);
+        } else {
+            tasks[taskCount] = new Task(line);
+        }
+    }
+
+    private void createTodo(String line) {
+        tasks[taskCount] = new Todo(line);
+    }
+
+    private void createEvent(String line) {
+        int index = line.indexOf("/at ");
+        String at = line.substring(index + 4);
+        line = line.substring(0, index - 1);
+        tasks[taskCount] = new Event(line, at);
+    }
+
+    private void createDeadline(String line) {
+        int index = line.indexOf("/by ");
+        String by = line.substring(index + 4);
+        line = line.substring(0, index - 1);
+        tasks[taskCount] = new Deadline(line, by);
+    }
+
+    public void listTasks() {
+        System.out.println("  ____________________________________________________________");
+        System.out.println("\tThese are the tasks in your list:");
+        for (int i = 0; i < taskCount; i += 1) {
+            int index = i + 1;
+            System.out.println("\t" + index + "." + tasks[i].getType() + tasks[i].getStatus()
```

Comment:
repetitive line that is long as well. can consider to create a new function or put it as toString method in the individual classes